generate-rss-feed.js

```diff
@@ -0,0 +1,14 @@
+import path from 'path'
+const __dirname = import.meta.dirname
+
+import { rssFeeder } from '@p-n-c/rssfeed'
+
+const src = path.join(__dirname, 'src')
+const options = {
+  root: 'src',
+  images: {
+    main: 'https://raw.githubusercontent.com/p-n-c/marketing/9b47dde787b5df57dd956277ca7154426ca2e80f/logo/PaC-logo.svg?token=AACAT4XXU6YMYIQWYMQK6CTHY5CNK',
```

Comment:
I get a 404 on the image and it doesn't show in the feed.

Reply:
Hm, I'm not sure why that is, it was working the other day. I'll review once we have images per article.

Reply:
Sure makes sense